Create a template for Inspur NF5180M6

sulik76 opened the pull request:
Create a template for Inspur NF5180M6 server platform.

danner26 commented:
Hello @sulik76 
VGA is not meant to be modelled as a console port. Please remove that
Please also add `inspur_` prefix to the `slug`. Thanks!

sulik76 commented:
Hello @danner26 
I corrected the template according to your instructions. I also added a comment with a link to the server's technical documentation.
